src/main/rx/srxl2.c

```diff
@@ -0,0 +1,545 @@
+#include "platform.h"
+
+#include "common/crc.h"
+#include "common/maths.h"
+#include "common/streambuf.h"
+
+#include "drivers/exti.h"
+#include "drivers/nvic.h"
+#include "drivers/time.h"
+
+#include "io/serial.h"
+
+#include "rx/srxl2.h"
+#include "rx/srxl2_types.h"
+#include "io/spektrum_vtx_control.h"
+
+#include "drivers/serial.h"
+#include "drivers/serial_uart.h"
+
+#include <string.h>
```

Comment:
System libraries go to the top, before `platform.h`.

Reply:
Done